.github/project.yml

```diff
@@ -1,4 +1,3 @@
 release:
   current-version: "schmalnau"
-  next-version: "lütter-SNAPSHOT"
-
+  next-version: "lütter-SNAPSHOT"
```

Comment:
This change needs to be reverted as well

Reply:
AHhh it must be an EOL.

Reply:
OK fixed!